source/command-logging-and-monitoring/command-logging-and-monitoring.rst

```diff
@@ -62,7 +62,7 @@ Publishing & Subscribing
 
 The driver SHOULD publish events in a manner that is standard to the driver's language publish/subscribe patterns and is not strictly mandated in this specification.
 
-Similarly, as described in the `logging specification <../logging/logging.rst#implementation-requirements>`_ the driver SHOULD emit log messages in a manner that is standard for the language.
+Similarly, as described in the `logging specification <../logging/logging.rst#implementation-requirements>`__ the driver SHOULD emit log messages in a manner that is standard for the language.
```

Comment:
The double underscore makes it anonymous and avoids the warning: `/Users/steve.silvester/workspace/specifications/source/server-discovery-and-monitoring/server-discovery-and-monitoring-logging-and-monitoring.rst:6: WARNING: Duplicate explicit target name: "logging specification"`